### DISCUSSION TITLE
Bug Report for Image Generation (Automatic1111)

### DISCUSSION THREAD

**Discussion by machine1111-code** [open-webui/open-webui#7538] [Issues]
````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````
[ERROR: HTTPConnectionPool(host='127.0.0.1', port=7860): Max retries exceeded with url: /sdapi/v1/sd-models (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5e82347cd0>: Failed to establish a new connection: [Errno 111] Connection refused'))]
````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````
Above error pops up upon trying to save settings with the following setup:
![Upload3](https://github.com/user-attachments/assets/92b84864-3ee2-415b-957f-8c3a90bd1f20)
![Error](https://github.com/user-attachments/assets/a4c093bb-a1f2-41f5-9bc1-f35b336638ae)
Copied instructions from this page: <https://docs.openwebui.com/features/images/>
Tried to run ollama on docker and outside of docker, neither worked, failure with same error.

Please help.

Thank you in advance.

**Comment by Bambouzle** [comment]
Hi,
I suppose you are running Open-WebUI inside docker. If so you need to use `http://host.docker.internal:7860`
Have a look at [the documentation](https://docs.openwebui.com/troubleshooting/network-diagrams) that helped me troubleshoot a similar issue I had.

**Reply by machine1111-code** [reply]
Thank you for response,

Do I just input ``````http://host.docker.internal:7860`````` into the box, just as is?

Will check the documentation.
